### PR TITLE
Add base aa1 to master

### LINKED ISSUE #1: The package.json should be updated
**Describe the feature or change you'd like**
The current package.json does not contain the basic details of this project.

**Additional context**
Add the basic information (name, version, homepage, ...) to the package.json.

### LINKED ISSUE #2: The code of conduct should be improved
**Describe the feature or change you'd like**
The current code of conduct is not meaningful in a way that represents our coding- and GitHub guidelines.

**Additional context**
Improve the code of conduct following this template:
Ref: https://github.com/aichner/Company-Homepage/blob/master/CODE_OF_CONDUCT.md

### LINKED ISSUE #3: The gitignore file should be adapted
**Describe the feature or change you'd like**
Currently, the gitignore does not exclude the MDB React Pro tar file, the DS_Store Apple file system files and the .env environment variable files.

**Additional context**
Update the gitignore to block those files.

### LINKED ISSUE #4: All components should be removed
**Describe the feature or change you'd like**
The current components are very obstructive when building a new project.

**Additional context**
Remove the current components. They will be added to a later date.

### LINKED ISSUE #6: The README is still a bit generic
**Describe the feature or change you'd like**
I would like to rework the README so it serves it's purpose.

**Additional context**
Ref: https://help.github.com/en/articles/about-readmes
Ref: https://github.com/snek-at/front-old/blob/master/README.md

### LINKED ISSUE #7: There should be a COPYING file
**Describe the feature or change you'd like**
Currently there is no file, that defines the license for copying the project. The copying file should be added to the root of the repository.

**Additional context**
- Ref: https://github.com/snek-at/intel/blob/master/COPYING

### LINKED ISSUE #8: The licenses should be updated to the latest state
**Describe the feature or change you'd like**
Currently this repository consist of the licenses from @Aichnerc, although the licenses should be changed to the license from @dermrsimon.

**Additional context**
- Ref: https://github.com/snek-at/client

### LINKED ISSUE #9: There should be import and export anchors
**Describe the feature or change you'd like**
Currently there are no import and exports anchors, they should be added.

**Additional context**
- Ref: https://github.com/snek-at/tonic/blob/master/STYLE_GUIDE.md

### LINKED ISSUE #10: There should be a basic components folder
**Describe the feature or change you'd like**
Currently there is no folder structure for basic components, the structure should be created.

**Additional context**
- Ref: 🐍

### LINKED ISSUE #11: The Visual Studio Code settings should be updated
**Describe the feature or change you'd like**
- The Visual Studio Code settings should be updated, so the tab length is 2 spaces.
- The Visual Studio Code settings should be updated, so the code is automatically formatted on saving.

**Additional context**
- Ref: https://github.com/snek-at/client/tree/master/.vscode

### PR DESCRIPTION
**Pull request by kleberbaum**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently package.json does not satisfy our needs. #1 
- Currently code of conduct does not satisfy our needs. #2 
- Currently `.gitignore` does not satisfy our needs. #3 
- Currently components are hindering our progress. #4  
- Currently the README is still a bit generic. #6 
- Currently there is no COPYING file. #7 
- Currently the licenses are not up to date. #8
- Currently there are no import and export anchors. #9
- Currently there is no basic components folder. #10
- Currently the Visual Studio code settings are outdated. #11

**What is the new behavior (if this is a feature change)?**
- Now package.json has been updated. Closes #1 
- Now code of conduct has been updated. Closes #2 
- Now `.gitignore` files has been updated. Closes #3 
- Now components have been removed. Closes #4 
- Now the README is no longer generic. Closes #6 
- Now there is a COPYING file. Closes #7 
- Now the licenses are up to date. Closes #8 
- Now there are import and export anchors. Closes #9 
- Now there is a basic components folder. Closes #10
- Now the Visual Studio Code settings are up to date. Closes #11

**Other information**:
- The version has been updated to v0.1.0-alpha to start this project.
- Ref: https://github.com/snek-at/front#versioning

### CODE REVIEW

**Review by kleberbaum — COMMENTED** [review]
Should be ok to merge.

**Review by petschna — APPROVED** [review]
Should be ok to merge.

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,42 +1,53 @@
...
-        "node_modules"
-    ],
-    "cSpell.words": [
-        "browserslist",
-        "homescreen",
-        "precached",
-        "renamer",
-        "webfonts"
-    ]
+  "editor.fontFamily": "Fira Code, Consolas, 'Courier New', monospace",
```
Comment (pinterid) [inline .vscode/settings.json:2]: Add these settings:
 - "prettier.trailingComma": "es5",
 - "prettier.arrowParens": "always",

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,9 +1,46 @@
...
+## [](#pull-requests)Pull Requests
+
+- Each pull request should only follow one topic
+- You must follow the guidelines / template
+- Each pull request must answer one more more issues
+- Each pull request must have at least one reviewer
+- Each pull request must have at least one assigned developers
+- Each pull request must have a label
+- Each pull request should be merged by a reviewer
+
```
Comment (pinterid) [inline CONTRIBUTING.md]: Remove newline.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,9 +1,46 @@
...
+  A code of conduct is a set of rules outlining the norms, rules, and responsibilities of, 
+  and or proper practices for, an individual.
+</p>
 
-Note we have a code of conduct (code of conflict), please follow it in all your interactions with the project.
+## Table of contents
+- [Table of contents](#table-of-contents)
+- [[](#commits)Commits](#commitscommits)
+- [[](#pull-requests)Pull Requests](#pull-requestspull-requests)
+
```
Comment (pinterid) [inline CONTRIBUTING.md]: Remove newline.

File: .vscode/settings.json (hunk 1 above)
Comment (pinterid) [reply .vscode/settings.json:2]: Just copy the .prettierrc file from intel. This will update the settings for prettier.

File: README.md [hunk 4]
```diff
@@ -99,7 +138,6 @@ to [contribute to our project](#contributing).
 ![GitHub repository license](https://img.shields.io/badge/license-EUPL--1.2-blue)
 
 SPDX-License-Identifier: (EUPL-1.2)
-Copyright © 2019 Werbeagentur Christian Aichner
+Copyright © Simon Prast
 
 [discord-badge]: https://img.shields.io/badge/Discord-Join%20chat%20%E2%86%92-738bd7.svg
```
Comment (pinterid) [inline README.md]: The badge should also be removed.

File: .vscode/settings.json (hunk 1 above)
Comment (schettn) [reply .vscode/settings.json:2]: I would suggest to implement this like intel does:
https://github.com/snek-at/intel/pull/77#discussion_r436252307

File: src/components/atoms/index.js [hunk 5]
```diff
@@ -1,16 +1,14 @@
-//> Components
+//#region > Imports
+//> Atoms
 // Import all components to export them for easy access from parent components
-import DocsLink from './DocsLink';
-import MenuLink from './MenuLink';
-import SectionContainer from './SectionContainer';
+//import Atom from "./Atom";
```
Comment (pinterid) [inline src/components/atoms/index.js:4]: Missing space after "//".

File: src/components/atoms/index.js [hunk 6]
```diff
@@ -1,16 +1,14 @@
...
+//#endregion
 
-export {
-    DocsLink,
-    MenuLink,
-    SectionContainer,
-};
+//#region > Exports
+//export { Atom };
```
Comment (pinterid) [inline src/components/atoms/index.js:8]: Missing space after "//".

File: src/components/molecules/index.js [hunk 7]
```diff
@@ -1,14 +1,14 @@
-//> Components
+//#region > Imports
+//> Molecules
 // Import all components to export them for easy access from parent components
-import Footer from './Footer';
-import Navbar from './Navbar';
+//import Molecule from "./Molecule";
```
Comment (pinterid) [inline src/components/molecules/index.js:4]: Missing space after "//".

File: package.json [hunk 8]
```diff
@@ -1,10 +1,11 @@
 {
-  "name": "react-mdb-template",
-  "version": "1.0.2",
+  "name": "snek-front",
+  "version": "1.0.0",
```
Comment (schettn) [inline package.json]: Wrong version number. Change it to:
```suggestion
  "version": "0.1.0",
```

File: src/components/molecules/index.js [hunk 9]
```diff
@@ -1,14 +1,14 @@
...
+//#endregion
 
-export {
-    Footer,
-    Navbar,
-};
+//#region > Exports
+//export { Molecule };
```
Comment (pinterid) [inline src/components/molecules/index.js:8]: Missing space after "//".

File: src/components/pages/index.js [hunk 10]
```diff
@@ -1,12 +1,14 @@
-//> Components
+//#region > Imports
+//> Pages
 // Import all components to export them for easy access from parent components
-import HomePage from './HomePage';
+//import Page from "./Page";
```
Comment (pinterid) [inline src/components/pages/index.js:4]: Missing space after "//".

File: src/components/pages/index.js [hunk 11]
```diff
@@ -1,12 +1,14 @@
...
+//#endregion
 
-export {
-    HomePage,
-};
+//#region > Exports
+//export { Page };
```
Comment (pinterid) [inline src/components/pages/index.js:8]: Missing space after "//".

File: package.json [hunk 12]
```diff
@@ -1,10 +1,11 @@
...
   "private": true,
+  "homepage": ".",
   "license": "(EUPL-1.2)",
   "dependencies": {
-    "mdbreact": "^4.19.1",
+    "mdbreact": "mdbreact-4.20.0.tgz",
```
Comment (schettn) [inline package.json]: This version is outdated. Change it to:
```suggestion
    "mdbreact": "mdbreact-4.26.1.tgz",
```

**Review by pinterid — CHANGES_REQUESTED** [review]
Looks good after some changes.

File: .gitignore [hunk 13]
```diff
@@ -7,9 +7,22 @@ package-lock.json
...
 
+# Keys
+.env
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+# MDB
+mdbreact-*
```
Comment (Aichnerc) [inline .gitignore]: Please change to `mdb*`. Some MDB ressources have names like `mdb-sortable-*`
```suggestion
mdb*
```

File: README.md [hunk 14]
```diff
@@ -1,34 +1,30 @@
 <p align="center">
-  <a href="https://www.aichner-christian.com/" target="_blank" rel="noopener noreferrer">
-    <img src="https://www.aichner-christian.com/img/logo/logo_web.png" alt="Agency Logo" height="150">
+  <a href="https://snek.at/" target="_blank" rel="noopener noreferrer">
+    <img src="https://avatars2.githubusercontent.com/u/55870326?s=400&u=c6c7f06305ddc94747d474850fde7b2044f53838&v=4" alt="Agency Logo" height="150">
```
Comment (Aichnerc) [inline README.md]: This is not the Agency logo.
```suggestion
    <img src="https://avatars2.githubusercontent.com/u/55870326?s=400&u=c6c7f06305ddc94747d474850fde7b2044f53838&v=4" alt="SNEK Logo" height="150">
```

File: CONTRIBUTING.md [hunk 15]
```diff
@@ -1,9 +1,46 @@
-# Contributing
+<p align="center">
+  <a href="https://www.aichner-christian.com/" target="_blank" rel="noopener noreferrer">
+    <img src="https://www.aichner-christian.com/img/logo/logo_web.png" alt="Agency Logo" height="150">
+  </a>
+</p>
```
Comment (schettn) [inline CONTRIBUTING.md]: There shouldn't be Aichner stuff.

**Review by schettn — CHANGES_REQUESTED** [review]
Please check

File: src/Routes.js [hunk 16]
```diff
@@ -1,23 +1,30 @@
...
-import React from 'react';
+import React from "react";
 // DOM bindings for React Router
-import { Route, Switch } from 'react-router-dom';
+import { Route, Switch } from "react-router-dom";
 
 //> Components
 /**
  * HomePage: A basic template page
  */
```
Comment (Aichnerc) [inline src/Routes.js]: This description is no longer needed, once it serves little purpose. It should be reworked with `@description` for each and every imported page.

File: src/Routes.js [hunk 17]
```diff
@@ -1,23 +1,30 @@
...
+/*
 import {
   HomePage
 } from './components/pages';
+*/
```
Comment (Aichnerc) [inline src/Routes.js]: I understand that this is the base, but to keep it clean please remove all unused imports.

File: src/Routes.js [hunk 18]
```diff
@@ -1,23 +1,30 @@
...
 class Routes extends React.Component {
   render() {
     return (
       <Switch>
-        <Route exact path='/' component={HomePage} />
-          
+        {
+          //<Route exact path='/' component={HomePage} />
+        }
+
```
Comment (Aichnerc) [inline src/Routes.js]: Remove unused code. This is not the proper way to declare Routes anyways.

File: CONTRIBUTING.md [hunk 19]
```diff
@@ -1,9 +1,46 @@
-# Contributing
+<p align="center">
+  <a href="https://snek.at/" target="_blank" rel="noopener noreferrer">
+    <img src="https://avatars2.githubusercontent.com/u/55870326?s=400&u=c6c7f06305ddc94747d474850fde7b2044f53838&v=4" alt="Agency Logo" height="150">
```
Comment (Aichnerc) [inline CONTRIBUTING.md]:
```suggestion
    <img src="https://avatars2.githubusercontent.com/u/55870326?s=400&u=c6c7f06305ddc94747d474850fde7b2044f53838&v=4" alt="SNEK Logo" height="150">
```

File: CODE_OF_CONDUCT.md [hunk 20]
```diff
@@ -1,15 +1,26 @@
-Code of Conflict
-----------------
+<p align="center">
+  <a href="https://www.aichner-christian.com/" target="_blank" rel="noopener noreferrer">
+    <img src="https://www.aichner-christian.com/img/logo/logo_web.png" alt="Agency Logo" height="150">
```
Comment (Aichnerc) [inline CODE_OF_CONDUCT.md]:
```suggestion
    <img src="https://www.aichner-christian.com/img/logo/logo_web.png" alt="SNEK Logo" height="150">
```

File: package.json [hunk 21]
```diff
@@ -13,11 +14,10 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/aichner/React-MDB-Template.git"
+    "url": "https://github.com/snek-at/front.git"
```
Comment (Aichnerc) [inline package.json]: Can't you use this fancy method here too?

```suggestion
    "url": "git+https://github.com/snek-at/front.git"
```

File: package.json [hunk 22]
```diff
@@ -13,11 +14,10 @@
...
   },
   "scripts": {
     "start": "react-scripts start",
-    "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject"
```
Comment (Aichnerc) [inline package.json]: Add required scripts.

```suggestion
    "build": "react-scripts build",
    "predeploy": "yarn run build",
    "deploy": "gh-pages -d build",
    "eject": "react-scripts eject"
```

File: CODE_OF_CONDUCT.md [hunk 23]
```diff
@@ -1,15 +1,26 @@
-Code of Conflict
-----------------
+<p align="center">
+  <a href="https://www.aichner-christian.com/" target="_blank" rel="noopener noreferrer">
```
Comment (pinterid) [inline CODE_OF_CONDUCT.md]: Remove aichner stuff from this file

File: src/components/atoms/index.js [hunk 24]
```diff
@@ -1,16 +1,14 @@
-//> Components
```
Comment (Aichnerc) [inline src/components/atoms/index.js:1]: There are no atoms yet. Please remove the folder `src/components/atoms` and re-add it as soon as you create atoms.

File: README.md (hunk 14 above)
Comment (pinterid) [inline README.md]: Change "Agency" to "SNEK".

File: CONTRIBUTING.md (hunk 19 above)
Comment (pinterid) [inline CONTRIBUTING.md]: Change "Agency" to "SNEK".

File: src/components/molecules/index.js [hunk 25]
```diff
@@ -1,14 +1,14 @@
-//> Components
```
Comment (Aichnerc) [inline src/components/molecules/index.js:1]: There are no molecules yet. Please remove the folder `src/components/molecules` and re-add it as soon as you create molecules.

File: src/components/pages/index.js [hunk 26]
```diff
@@ -1,12 +1,14 @@
-//> Components
```
Comment (Aichnerc) [inline src/components/pages/index.js:1]: There are no pages yet. Please remove the folder `src/components/pages` and re-add it as soon as you create pages.

**Review by Aichnerc — CHANGES_REQUESTED** [review]
There are a few things I've noticed. 🧶

File: .prettierrc [hunk 27]
```diff
@@ -0,0 +1,6 @@
+{
+  "trailingComma": "es5",
+  "arrowParens": "always",
+  "tabWidth": 2,
+  "singleQuote": false,
+}
```
Comment (schettn) [inline .prettierrc]: Please add a newline:
```suggestion
}

```

File: package.json [hunk 28]
```diff
@@ -13,12 +14,13 @@
   },
   "repository": {
     "type": "git",
```
Comment (Aichnerc) [inline package.json:16]: Is the type still required if you include `git+` in the url?

**Review by schettn — APPROVED** [review]
LGTM 🚀

File: src/App.js [hunk 29]
```diff
@@ -1,41 +1,48 @@
...
+import React from "react";
 // DOM bindings for React Router
-import { BrowserRouter as Router } from 'react-router-dom';
+import { BrowserRouter as Router } from "react-router-dom";
 
 //> Components
 /**
  * Footer: Global Footer
  * Navbar: Global navigation bar
  */
```
Comment (Aichnerc) [inline src/App.js:12]: Consider using `@description`

**Review by pinterid — APPROVED** [review]
Looks good.

File: src/Routes.js [hunk 30]
```diff
@@ -1,23 +1,19 @@
...
+import { Route, Switch } from "react-router-dom";
 
 //> Components
-/**
- * HomePage: A basic template page
- */
-import {
-  HomePage
-} from './components/pages';
+// import { SamplePage } from "./components/pages";
```
Comment (Aichnerc) [inline src/Routes.js:9]: Consider creating a sample `@description` to make it easier to not forget to add documentation later.

**Review by Aichnerc — APPROVED** [review]
Feel free to look at my suggestions, but the code looks ok nevertheless.

File: src/components/atoms/index.js (hunk 24 above)
Comment (pinterid) [reply src/components/atoms/index.js:1]: We preferred to create the structure with this branch, to make further contributing easier.

File: src/components/molecules/index.js (hunk 25 above)
Comment (pinterid) [reply src/components/molecules/index.js:1]: We preferred to create the structure with this branch, to make further contributing easier.

File: src/components/pages/index.js (hunk 26 above)
Comment (pinterid) [reply src/components/pages/index.js:1]: We preferred to create the structure with this branch, to make further contributing easier.

**Pull request merged**